packages/Panel/src/components/Header/Header.js

```diff
@@ -23,52 +23,47 @@ const Header = React.forwardRef((props, ref) => {
     <sc.Header
       data-pka-anchor="panel.header"
       hasPushedElement={!!getPushContentElement}
-      isCompact={isCompact}
-      kind={kind}
+      size={size}
+      hasAccent={hasAccent}
       ref={ref}
       {...moreProps}
     >
-      <Heading level={level} displayLevel={isCompact ? 4 : 3} ref={refHeading}>
+      <Heading level={level} ref={refHeading}>
         {children}
       </Heading>
       {hasCloseButton && (
         <Button.Close
           data-pka-anchor="panel.header.close"
           isSemantic={false}
           onClick={onClose}
-          size={isCompact ? types.SMALL : types.MEDIUM}
-          isDark={kind === types.kinds.PRIMARY}
+          size="medium"
         />
```

Comment:
Prettier issue, change it to
`<Button.Close data-pka-anchor="panel.header.close" isSemantic={false} onClick={onClose} size="medium" />`